docs/ConfigurationFile.md

```diff
@@ -47,6 +47,24 @@ validation:
     iou_threshold: 0.4
     val_accuracy_interval: 20
 ```
+## Passing config arguments at runtime using a dict
+
+It can often be useful to pass config args directly to a model instead of editing the config file. By using a dict with that matches with the config keys, main.deepforest will update the config after reading from file.
```

Comment:
I found the second sentence confusing. How about:

"By using a dict containing the config keys and their values. Values provided in this dict will override values provided in deepforest_config.yml."